howto/cluster/scale-down.md

```diff
@@ -1,24 +1,38 @@
 Scaling down a LXD cluster involves more checks than scaling up.
 
-First, you must pick a suitable candidate node. When a node is removed, all still running containers on it are stopped and potentially connected users are disconnected. To avoid this, AMS provides a feature to mark a node as unschedulable so that it will not be considered for any further container launches.
+There are two important things to know when scaling down:
+ - The node you remove **MUST NOT** have any container left.
+ - You **MUST** wait for a node to be fully removed before you can start removing another one
 
+1. Preparing the node for removal
+
+First, pick the node you want to remove.
+We have to tell AMS to stop considering this node for new containers.
 You can mark a node as unschedulable with the following command:
 
     amc node set lxd0 unscheduable true
 
 (Note that the typo in the command will be fixed in a future Anbox Cloud release.)
 
-Now the node won't be considered for any further container launches. As it might still host containers, you can decide to either kill all containers or wait for your users to disconnect. You can check with the following command if the node still hosts containers:
+Now the node won't be considered for any further container launches.
+
+The node might still have containers running, you can decide to either kill all containers or wait for your users to disconnect. You can check with the following command if the node still hosts containers:
 
     amc ls --filter node=lxd0 --format=csv | wc -l
 
+Note that a node **MUST** have no container left on it before you can remove it.
+
 If you want to kill all containers immediately, run the following command:
 
-    for id in $(amc ls --filter node=lxd0 --format=csv | cut -d, -f1) ; do amc delete -y "$id" ; done
+    amc delete --all --yes
+
+2. Removing the node
+
+When the node is ready to be removed, use the following Juju command:
 
-When the node is ready to be removed, remove it by using Juju:
+    juju remove-unit lxd/<id>
 
-    juju remove-unit lxd/0
+where `<id>` is the number of the unit you are removing
```

Comment:
Move this to the intro ("When the node is ready to be removed, use the following Juju command, where `<id>` is the number of the unit you are removing:")